#! /bin/sh /usr/share/dpatch/dpatch-run
## 10_charset_fixes.dpatch by Daniel T Chen <crimsun@fungus.sh.nu>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Additional charset includes from changeset 12758

@DPATCH@

--- trunk/modules/gui/wxwidgets/interface.cpp	(revision 12757)
+++ trunk/modules/gui/wxwidgets/interface.cpp	(revision 12758)
@@ -29,6 +29,7 @@
 #include <vlc/vout.h>
 #include <vlc/input.h>
 #include <vlc/intf.h>
+#include "charset.h"
 
 #include "wxwidgets.h"
 
--- trunk/modules/gui/wxwidgets/open.cpp	(revision 12757)
+++ trunk/modules/gui/wxwidgets/open.cpp	(revision 12758)
@@ -31,6 +31,7 @@
 #include <stdio.h>
 
 #include <vlc/vlc.h>
+#include "charset.h"
 
 #ifdef HAVE_LIBCDIO
 #include <cdio/cdio.h>
--- trunk/modules/gui/wxwidgets/dialogs.cpp	(revision 12757)
+++ trunk/modules/gui/wxwidgets/dialogs.cpp	(revision 12758)
@@ -32,6 +32,7 @@
 #include <vlc/vlc.h>
 #include <vlc/aout.h>
 #include <vlc/intf.h>
+#include "charset.h"
 
 #include "wxwidgets.h"
 
